Configs/.config/fish/config.fish

```diff
@@ -4,12 +4,28 @@ if status is-interactive
     starship init fish | source
 end
 
-# List Directory
-alias ls="lsd"
-alias l="ls -l"
-alias la="ls -a"
-alias lla="ls -la"
-alias lt="ls --tree"
+
+
+#for btop
+export LANG=en_US.UTF-8
+export LC_ALL=en_US.UTF-8
+
+
+# Cleanup local orphaned packages
+function cleanup
+    while pacman -Qdtq
+        sudo pacman -R (pacman -Qdtq)
+    end
+end
```

Comment:
this may cause the removal of wanted packages
in my case packages like these were also listed
![image](https://github.com/prasanthrangan/hyprdots/assets/94917588/db353195-5d54-4739-aab3-12f6a42d8056)

Reply:
Ok then i should make this feature as optional and i will comment it on fish config

Reply:
> this may cause the removal of wanted packages in my case packages like these were also listed ![image](https://private-user-images.githubusercontent.com/94917588/313487481-db353195-5d54-4739-aab3-12f6a42d8056.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA4NjYzNzEsIm5iZiI6MTcxMDg2NjA3MSwicGF0aCI6Ii85NDkxNzU4OC8zMTM0ODc0ODEtZGIzNTMxOTUtNWQ1NC00NzM5LWFhYjMtMTJmNmE0MmQ4MDU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE5VDE2MzQzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE3ZjQ3OTQ2NWJmM2ZkOWZlYmU5YTAxNDYxMTZkOWUzOGQ0Njc0MDk1NWI5NDIxN2ExNDJiMjU3N2YwOWU0MWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RvOlpZXB-yA3Tj46w0O0YMx2QNIR4oRkIUuh2D6V3Mk)

about this issue please check [this](https://github.com/ALEX5402/hyprdots/commit/c43a7e040a9d25dff0f60538527c1dca2d5c1a32#diff-f63351d384d0e6c52749b4f21daa510d5f613dc0b941493d03833d84824263c1) commit